packages/pwa-kit-dev/src/utils/glob.test.js

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright (c) 2021, salesforce.com, inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-import assert from 'assert'
-import {Matcher} from './glob'
-
-test('Matcher with no patterns matches nothing', () => {
-    const matcher = new Matcher()
-    expect(matcher.matches('')).toBe(false)
-    expect(matcher.matches('a.js')).toBe(false)
-    expect(matcher.matches()).toBe(false)
-})
-
-describe('Matcher class filters correctly', () => {
-    const patterns = ['ssr.js', '**/*.jpg', '!**/no.jpg', 'abc.{js,jsx}']
-
-    const matcher = new Matcher(patterns)
-
-    // Paths we expect to match
-    const expectToMatch = [
-        'ssr.js',
-        'test1.jpg',
-        'static/test2.jpg',
-        'static/assets/test3.jpg',
-        'abc.js',
-        'abc.jsx'
-    ]
-
-    expectToMatch.forEach((path) =>
-        test(`Expect path "${path}" to match`, () => {
-            assert.ok(matcher.matches(path), `Expected path "${path}" to be matched`)
-        })
-    )
-
-    // Paths we expect not to match
-    const expectNotToMatch = ['ssrxjs', 'subdirectory/ssr.js', 'no.jpg', 'static/no.jpg', 'abc.jsz']
-
-    expectNotToMatch.forEach((path) =>
-        test(`Expect path "${path}" to NOT match`, () => {
-            assert.ok(!matcher.matches(path), `Expected path "${path}" to NOT be matched`)
-        })
-    )
-
-    // Combine the paths into one array and shuffle it
-    const allPaths = expectToMatch.concat(expectNotToMatch).sort(() => Math.random() - 0.5)
-    test('Matcher.filter works', () => {
-        const matched = allPaths.filter(matcher.filter)
-        assert.strictEqual(
-            matched.length,
-            expectToMatch.length,
-            'Expected that all matches would be returned by filter'
-        )
-    })
-})
```

Comment:
Do we use the `glob` library anywhere else? Do we want to remove it from `package.json`? Funny story "Hey, I was using that" (e.g. the `glob` lib) in some experimental work, but I see `fast-glob` is also used in the bundle as a child dependency of some typescript compiler or something so I'm not worried. 

But yeah, we can remove if no longer using

Reply:
It's still in use, it's just hard to see through the refactoring. I'd love to get rid of it, too, but it's a hard requirement of our bundle-upload API at this point.

There is a discussion around refactoring that endpoint, but it's taking a while!